vfs/vfscache/cache.go

```diff
@@ -511,6 +571,18 @@ func (c *Cache) clean() {
 	// oldest first
 	c.purgeOverQuota(int64(c.opt.CacheMaxSize))
 
+	// removeCleanFiles indicates that we got ENOSPC error
```

Comment:
In our use case, the cache size will be relatively small and it will be filled quickly, like in a few seconds.  So we probably want delete all clean files so that the cleaner does not get kicked even more quickly to repeat the cache reset process.  I think we can achieve this by setting a very low quota in our use case and allow other use cases to reset just enough clean cache files to make the total cache space usage to be below the quota.  I will add the quota check in purgeClean().